drivers/pinmux/pinmux_sam0.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2018 Google LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <pinmux.h>
+#include <soc.h>
+
+struct pinmux_sam0_config {
+	PortGroup *regs;
+};
+
+static int pinmux_sam0_set(struct device *dev, u32_t pin, u32_t func)
+{
+	const struct pinmux_sam0_config *cfg = dev->config->config_info;
+
+	if ((pin & 1) == 0) {
```

Comment:
Can we get rid of the magic numbers here?

Reply:
Done.  Added a comment explaining the packing and pulled the args out to variables.